Cargo.toml

```diff
@@ -46,3 +51,12 @@ members = [
   "./examples/hello-world/crate",
   "./examples/input-form/crate",
 ]
+
+[patch.crates-io]
```

Comment:
I find the large array of patches here quite amusing, although arguably a bug in Cargo so perhaps less so...